app/javascript/packs/show_image.js

```diff
@@ -0,0 +1,18 @@
+$('#input').on('change', function() {
+    showImage(this);
+})
+
+function showImage(input) {
+    if (input.files && input.files[0]) {
+        let reader = new FileReader();
+
+        reader.onload = function (e) {
+            $('#your_preview_id')
+                .attr('src', e.target.result)
+                .width(500)
+                .height(200);
```

Comment:
can we remove these hardcode sizes? and we need to save image proportions, now they are hardcoded even if you will load image with 4:3 ratio, make it full width for the container, and height auto

Reply:
Also, can we detect image extensions? now I can load any file and it will try to make a preview, and it raises 500 error when you try to load non-image file